test/data/subnet/subnet_test_proc.txt

```diff
@@ -0,0 +1,40 @@
+Time           value  port                component
```

Comment:
Time Value Port Component

Heading of the outputs should be in a similar format. (Either all Uppercase or lowercase). If not the first alphabet in uppercase for all columns in headings

Reply:
solved